### ISSUE TITLE
Adding sub menu with header not shows selected item effect for sub items

### ISSUE THREAD

**Issue by ShreyashPromact** [smuldr/design-support-demo#1]
Hello,
I have refer this android developer blog for the NavigationView.
http://android-developers.blogspot.in/2015/05/android-design-support-library.html

I have updated your drawer.xml layout as below:

<?xml version="1.0" encoding="utf-8"?>
<menu xmlns:android="http://schemas.android.com/apk/res/android">
  
  <group android:checkableBehavior="single">
    <item
      android:id="@+id/drawer_item_1"
      android:icon="@drawable/ic_info"
      android:title="@string/item_1"/>
    <item
      android:id="@+id/drawer_item_2"
      android:icon="@drawable/ic_help"
      android:title="@string/item_2"/>
    <item
      android:id="@+id/navigation_subheader"
      android:title="Some title">
      <menu>
        <item
          android:id="@+id/navigation_sub_item_1"
          android:icon="@drawable/ic_info"
          android:title="option 1"/>
        <item
          android:id="@+id/navigation_sub_item_2"
          android:icon="@drawable/ic_help"
          android:title="option 2"/>
      </menu>
    </item>
  </group>
</menu>

Here, I have added more menu items with header. But after that when i click on item that are in Sub menu, it is not showing highlighted effect. But it is working fine for top two items.

Where it need to update. I guess there is nothing to update in the Java file.  If it is then please let me know.

Thanks for this example.

Sincerely,
Shreyash

**Comment by smuldr** [comment]
I don't think there is an easy solution for this. Highlighting submenu items is one of the open issues in the current version of the NavigationView, as I also mentioned in the blog post. It seems the NavigationView is really meant for very basic applications. If you want fancy things like submenu item highlighting, you will still need to implement your own navigation view.

**Comment by ShreyashPromact** [comment]
No. It is clear mention in the google blog that we can add header and sub menu in that way. I guess there should not be the custom implementation for that effect. 

I suggest to please keep open this issue until google or other contributor not found proper solution to this. There has to be some solution to this.

Thanks for quick reply.

Sincerely,
Shreyash
